interface/ceed.c

```diff
@@ -636,6 +643,22 @@ int CeedSetData(Ceed ceed, void* *data) {
   return 0;
 }
 
+/**
+  @brief Get the full resource name for a CEED
+
+  @param ceed            Ceed to get resource name of
+  @param[out] resource   Variable to store resource name
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Basic
+**/
+
+int CeedGetResource(Ceed ceed, char* *resource) {
```

Comment:
Should not return a mutable pointer. Also, `char **resource` is just fine. No need to mix syntax conventions just because it's a `**`.

Reply:
In this case would it be typical to allocate the space, make a copy, and expect the user to free it? I thought about that but wasn't sure.

We've done `char* *whatever` in a few places, so I'll change it globally.

Reply:
I don't object to returning a pointer to `const`, it just needs to be `const`.

Reply:
Fixed